src/domains/shared-app/03_influxdb.tf

```
@@ -0,0 +1,35 @@
+# influxdb v2
+resource "helm_release" "influxdb2" {
+  name = "pagopa-influxdb2"
+
+  repository = "https://helm.influxdata.com/"
+  chart      = "influxdb2"
+  version    = var.influxdb2_helm_version
+  namespace  = kubernetes_namespace.namespace.metadata[0].name
+
+  set {
+    name  = "service.type"
+    value = "ClusterIP"
+  }
+
+  set {
+    name  = "ingress.enabled"
+    value = true
+  }
+
+  set {
+    name  = "ingress.tls"
+    value = true
+  }
+
```

```suggestion
  set {
    name  = "image.tag"
    value = var.influxdb2_helm_image_tag
  }

  set {
    name  = "image.repository"
    value = "influxdb"
  }
```

To prevent supply chain attack, it's better to use docker image tag with sha256

https://hub.docker.com/layers/library/influxdb/2.2.0-alpine/images/sha256-f3b54d91cae591fc3fde20299bd0b262f6f6d9a1f73b98d623b501e82c49d5fb?context=explore
```
var.influxdb2_helm_image_tag = "influxdb:2.2.0-alpine@sha256:f3b54d91cae591fc3fde20299bd0b262f6f6d9a1f73b98d623b501e82c49d5fb"
```

---

done